25837.6 <c-34#153> Created logger
25837.7 <c-34#153> appl_initialize was called
25837.7 <c-34#153:2> Logger was updated
25837.7 <c-34#153:2> Started intializing cube lib
25837.7 <c-34#153:2> Finished intializing cube lib with worker, starting to work
25838.4 <c-34#153:2> requestCubes => [46] Buffer size: 0
25838.4 <c-34#153:2> Sent requestCubes signal to root
25838.4 <c-34#153:2> Received 1 cubes from root
25838.4 <c-34#153:2> The main loop continues
25838.4 <c-34#153:2> Started solving a cube
25838.4 <c-34#153:2> Cube failed
25838.4 <c-34#153:2> Added failed cube
25838.4 <c-34#153:2> returnFailedCubes => [46] Buffer size: 3
25838.4 <c-34#153:2> Sent 1 failed cubes to root
25838.4 <c-34#153:2> Received receivedFailedCubes signal from root
25838.4 <c-34#153:2> requestCubes => [46] Buffer size: 0
25838.4 <c-34#153:2> Sent requestCubes signal to root
25838.4 <c-34#153:2> Received 1 cubes from root
25838.4 <c-34#153:2> The main loop continues
25838.4 <c-34#153:2> Started solving a cube
26808.9 <c-34#153:2> appl_interrupt was called
26808.9 <c-34#153:2> Interrupting worker
26808.9 <c-34#153:2> appl_interrupt was called
26808.9 <c-34#153:2> appl_withdraw was called
26808.9 <c-34#153:2> Started withdraw thread
26808.9 <c-34#153:2> STOP (0.00s since last cb)
26808.9 <c-34#153:2> Solving interrupted
26808.9 <c-34#153:2> The main loop continues
26808.9 <c-34#153:2> Leaving the main loop due to an interruption
26808.9 <c-34#153:2> Finished withdrawing lib
26808.9 <c-34#153:2> Enter destructor of CubeWorker
26808.9 <c-34#153:2> Time waiting for messages: 0.000
26809 <c-34#153:2> Enter destructor of CubeWorkerInterface
26809 <c-34#153:2> Enter destructor of CubeRoot
26809 <c-34#153:2> Finished withdraw thread
26871 <c-34#153:2> Enter destructor
26871 <c-34#153:2> Time working: 971.225
26871 <c-34#153:2> Time suspended: 0.000
26871 <c-34#153:2> Joined cleanup thread
26871 <c-34#153:2> Exit destructor
26871 <c-34#153:2> Destructing logger
